src/commands/selectSignImage.ts

```diff
@@ -0,0 +1,142 @@
+import * as vscode from 'vscode';
+import { logInToDockerCli } from './registries/logInToDockerCli';
+import { checkCli } from '../utils/checksCli';
+import { execAsync } from '../utils/execAsync';
+
+const cliName = 'notation';
+const goToNotationButton = 'Download Notation';
+const NotationURL = 'https://notaryproject.dev/docs/installation/cli/';
+const errorMessage = `Notation not found: `;
+
+
+/**
+ * Parsing the string is necessary because the current notation output
+ * is a tabular string, thus not formatted for immediate use. 
+ * Thus the function changes the key list string into an array of objects
+ */
+export async function sortOutput(Key_String_Values: string ){
+    //if the key_string_values is empty
+    if(Key_String_Values == ""){
+        return "No current keys in notation";
+    }else {
+        var dataArray = new Array();
+        var defaultKey = '';
+        var keyChoices = Key_String_Values.split("\n");
+        for (let i = 1; i < keyChoices.length; i++) { 
```

Comment:
When doing these sorts of parsing I recommend including examples in comments of what it is you are parsing. This provides a clear understanding of what it is you are expecting as an input. This function is also a great candidate for unit tests.